packages/core/src/Primitive/usePrimitiveElement.test.ts

```
@@ -0,0 +1,33 @@
+import { mount } from '@vue/test-utils'
+import { describe, expect, it } from 'vitest'
+import { defineComponent, h } from 'vue'
```

ESLint

[error] 'h' is defined but never used.

(unused-imports/no-unused-imports)